sled-agent/Cargo.toml

```diff
@@ -17,9 +17,12 @@ crucible-agent-client = { git = "https://github.com/oxidecomputer/crucible", rev
 dropshot = { git = "https://github.com/oxidecomputer/dropshot", branch = "main", features = [ "usdt-probes" ] }
 futures = "0.3.21"
 ipnetwork = "0.18"
+libc = "0.2.123"
 macaddr = { version = "1.0.1", features = [ "serde_std" ] }
 nexus-client = { path = "../nexus-client" }
 omicron-common = { path = "../common" }
+opte-ioctl = { git = "https://github.com/oxidecomputer/opte", rev = "2df10f2" }
+opte = { git = "https://github.com/oxidecomputer/opte", rev = "2df10f2", features = [ "api", "std", "engine" ] }
```

Comment:
Hmm, I'm not sure I understand the features in that case. I believe I only added that _after_ failing to compile. But I'll definitely try again, thanks!

Reply:
Confirmed this feature isn't needed.

Reply:
Yea, the reason you needed this before is because you were pulling `Vni` from engine. To be fair it's not like there are any great docs around this at the moment. But just for future reference, everything in `engine` is meant to be internal to the implementation. Any client of OPTE should only ever have to see the `api` namespace.